diff -up netcdf-fortran-4.4.1/configure.postdeps netcdf-fortran-4.4.1/configure
--- netcdf-fortran-4.4.1/configure.postdeps	2014-09-16 21:40:02.181108003 -0600
+++ netcdf-fortran-4.4.1/configure	2014-09-16 21:40:58.470207376 -0600
@@ -16205,10 +16205,11 @@ if { { eval echo "\"\$as_me\":${as_linen
     case ${prev}${p} in
 
     -L* | -R* | -l*)
-       # Some compilers place space between "-{L,R}" and the path.
+       # Some compilers place space between "-{L,R,l}" and the path.
        # Remove the space.
        if test $p = "-L" ||
-          test $p = "-R"; then
+          test $p = "-R" ||
+          test $p = "-l"; then
 	 prev=$p
 	 continue
        fi
