diff -up netcdf-fortran-4.2/configure.mpi netcdf-fortran-4.2/configure
--- netcdf-fortran-4.2/configure.mpi	2012-09-06 15:31:03.123237745 -0600
+++ netcdf-fortran-4.2/configure	2012-09-06 15:34:13.450564441 -0600
@@ -12198,9 +12198,9 @@ $as_echo "$enable_static" >&6; }
     ## There is no encapsulation within the following macros, do not change
     ## the running order or otherwise move them around unless you know exactly
     ## what you are doing...
-    lt_prog_compiler_wl_F77=
-lt_prog_compiler_pic_F77=
-lt_prog_compiler_static_F77=
+    lt_prog_compiler_wl_F77='-Wl,'
+lt_prog_compiler_pic_F77='-fPIC'
+lt_prog_compiler_static_F77='-static'
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for $compiler option to produce PIC" >&5
 $as_echo_n "checking for $compiler option to produce PIC... " >&6; }
@@ -14902,9 +14902,9 @@ fi
 
 
 
-    lt_prog_compiler_wl_FC=
-lt_prog_compiler_pic_FC=
-lt_prog_compiler_static_FC=
+    lt_prog_compiler_wl_FC='-Wl,'
+lt_prog_compiler_pic_FC='-fPIC'
+lt_prog_compiler_static_FC='-static'
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for $compiler option to produce PIC" >&5
 $as_echo_n "checking for $compiler option to produce PIC... " >&6; }
