--- netcdf-fortran-4.2-beta2/nf-config.in.pkgconfig	2011-08-18 10:17:58.000000000 -0600
+++ netcdf-fortran-4.2-beta2/nf-config.in	2011-08-29 10:46:20.890471535 -0600
@@ -8,7 +8,6 @@
 
 prefix=@prefix@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
 includedir=${prefix}/include
 
 cc="@CC@"
@@ -16,8 +15,8 @@
 fc="@FC@"
 cflags=" -I${includedir} @CPPFLAGS@" 
 fflags="@FFLAGS@ @MOD_FLAG@${includedir}"
-libs="-L${libdir} @NC_LIBS@"
-flibs="-L${libdir} @NC_FLIBS@"
+libs=`pkgconfig netcdf-fortran --libs`
+flibs=`pkgconfig netcdf-fortran --flibs`
 has_dap="@HAS_DAP@"
 has_nc2="@HAS_NC2@"
 has_nc4="@HAS_NC4@"
@@ -141,11 +140,11 @@
 	;;
 
      --libs)
-       	echo $libs
+       	pkgconfig netcdf-fortran --libs
        	;;
 
     --flibs)
-       	echo $flibs
+       	pkgconfig netcdf-fortran --flibs
        	;;
 
     --has-dap)
